index.js

```diff
@@ -33,9 +39,9 @@ module.exports = function(glob, options) {
 
   // make glob absolute
   if (rootDir && glob.charAt(0) === '/') {
-    glob = path.join(rootDir, glob);
-  } else if (!isAbsolute(glob)) {
-    glob = path.resolve(cwd, glob);
+    glob = join(rootDir, glob);
+  } else if (process.platform === 'win32' || !isAbsolute(glob)) {
```

Comment:
I think this is the last remaining problem.  Since globs aren't supposed to have `\\` in them we should be passing something like `C:/test/123` for our glob, but `isAbsolute` returns false for that.